content/docs/user-guide/networking/multiplayer/spawning.md

```diff
@@ -5,7 +5,9 @@ description: A reference for spawning and registering network player entities in
 weight: 700
 ---
 
-In most scenarios, when a player joins a multiplayer session, a controllable networked entity must be spawned for the player. The **Multiplayer Gem** in **Open 3D Engine (O3DE)** provides an interface to specify an *autonomous* entity for a player. For more information about autonomous roles, refer to the [Multiplayer Gem Overview](overview#multiplayer-entity-roles).
+In most networking multiplayer games, when a player joins a session, the host must spawn an [*autonomous*](/docs/user-guide/networking/multiplayer/overview.md#multiplayer-entity-roles) networked entity for the player. Likewise, when a player leaves, the host must remove the entity and clean up. In **Open 3D Engine (O3DE)**, you can set this up exactly, or handle join and leave events in other ways by using `OnPlayerJoin` and `OnPlayerLeave` events, which are provided by the **Multiplayer Gem**'s `IMultiplayerSpawner` interface. Alternatively, to get started sooner, you can bypass programming these events and use the **Simple Player Spawner** component, which sets up `OnPlayerJoin` and `OnPlayerLeave` for this common use case.
```

Comment:
Add link to simple spawner component.
Also, nit: use relative link here.
```suggestion
In most networking multiplayer games, when a player joins a session, the host must spawn an [*autonomous*](overview#multiplayer-entity-roles) networked entity for the player. Likewise, when a player leaves, the host must remove the entity and clean up. In **Open 3D Engine (O3DE)**, you can set up your spawning logic in this way, or handle join and leave events in other ways by using `OnPlayerJoin` and `OnPlayerLeave` events, which are provided by the **Multiplayer Gem**'s `IMultiplayerSpawner` interface. Alternatively, to get started sooner, you can bypass programming these events and use the [**Simple Player Spawner**](/docs/user-guide/components/reference/multiplayer/simple-player-spawner) component, which sets up `OnPlayerJoin` and `OnPlayerLeave` for this common use case.
```

Reply:
^ Updated to "Simple Network Player Spawner"